### DISCUSSION TITLE
openwrt 连接报错404

### DISCUSSION THREAD

**Discussion by rq1025330** [messense/aliyundrive-webdav#65] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/52870887/136337475-92242beb-7f66-431f-bca6-e289d67dc364.png)

**Comment by messense** [comment, marked as answer]
根目录参数是云盘目录不是本地磁盘路径，除非需要限制浏览的文件夹，一般用默认的 `/` 就行。

**Comment by messense** [comment]
更新到 0.4.2 版本后界面有说明，https://github.com/messense/aliyundrive-webdav/commit/eb3af903432736dbe0b2c615a6f9409bc491c0b8

**Comment by rq1025330** [comment]
明白了 ，谢谢
